index.html

```diff
@@ -63,28 +56,20 @@
         <div class="col-md-4">
             <i class="top3-icon fa fa-thumbs-up"></i>
 
-            <p class="strong-heading">Simple</p>
-            <p>Easy to setup, simple to master. Flyway lets you regain control of your database migrations with pleasure
-                and plain sql.</p>
-            <a class="btn btn-primary" href="/documentation/getstarted/why">Why database migrations? <i
-                    class="fa fa-arrow-right"></i></a>
+            <p class="strong-heading">Simple to use</p>
```

Comment:
Could we force this to be in two lines / match the amount of space the other two headings in this row take? It looks weird having the text be misaligned

![image](https://user-images.githubusercontent.com/17652619/95313419-0658f600-0888-11eb-943b-3ce910770a89.png)